_docs/concepts/load-balancing.md

```diff
@@ -0,0 +1,33 @@
+---
+category: Concepts
+title: Discovery & Load Balancing
+overview: Describes how traffic is load balanced across instances of a service in the mesh.
+              
+parent: Traffic Management
+order: 25
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+Istio relies on the platform to automatically register pods/containers to
```

Comment:
I think we should weaken and generalize this. When Istio is running on bare HW or straight VMs, there is no omnipresent guardian that takes care of this stuff, humans will have to manually provision this kind of state.